scripts/mteb_to_tex.py

```diff
@@ -0,0 +1,225 @@
+"""
```

Comment:
Seems like it might be worth creating a CLI for some of this, which we can make sure is also tested (such that future edits also make sure to fix it)

Something like:
```
mteb create_table style=tex folder={path_to_folder} benchmark="mteb_en"  
```